prime-angular-frontend/src/app/shared/components/college-certification-form/college-certification-form.component.ts

```diff
@@ -122,8 +124,19 @@ export class CollegeCertificationFormComponent implements OnInit {
       });
 
     this.licenseCode.valueChanges
-      .subscribe((licenseCode: number) =>
+      .subscribe((licenseCode: number) => {
+
+        const matchingLicenses = this.licenses.filter(licenseConfig => licenseConfig.code === licenseCode);
+        if (matchingLicenses.length != 1) {
```

Comment:
Could probably reduce this to just a if and else. We can assume that we will always have a match since they are being populated by the same list. So you can just check on the validate (you can add a '?' to handle the fluke chance it didn't return 1 value)

```
if ((matchingLicenses[0] as LicenseWeightedConfig)?.validate) {
...
} else {
...
}
```